packages/react-vapor/src/components/editor/CodeEditor.tsx

```diff
@@ -38,8 +42,15 @@ const mapStateToProps = (state: IReactVaporState, {collapsibleId}: ICodeEditorPr
     isCollapsibleExpanded: CollapsibleSelectors.isExpanded(state, collapsibleId),
 });
 
+const mapDispatchToProps = (dispatch: IDispatch, {id}: {id?: string}) => ({
+    updateStoreValue: _.debounce((value: string) => {
+        dispatch(CodeEditorActions.updateValue(id, value));
+    }, 500),

```

Comment:
I think this will create a new debounced function each time the state changes, but I might be mistaken. A safer approach is to debounce the prop from inside the component's implementation.

Reply:
ok I'll do that